app/controllers/concerns/api_helper.rb

```diff
@@ -82,7 +91,7 @@ def valid_token?(token)
   def post_req(uri, body)
     http = Net::HTTP.new(uri.host, uri.port)
     http.use_ssl = (uri.scheme == 'https')
-    exp = Time.now.to_i + 24 * 3600
+    exp = Time.now.to_i + (24 * 3600)
```

Comment:
Unrelated change?

Reply:
Got caught by rubocop. If it is not changed it won't pass